hack/lib/etcd.sh

```diff
@@ -36,12 +36,12 @@ kube::etcd::validate() {
 
   # validate installed version is at least equal to minimum
   version=$(etcd --version | tail -n +1 | head -n 1 | cut -d " " -f 3)
-  if [[ "${version}" < "${ETCD_VERSION}" ]]; then
+  if [[ "`echo -e "${version}\n${ETCD_VERSION}" | sort -rV | head -n 1`" != "$version" ]]; then
```

Comment:
`V` is a gnu extension to sort :-/

http://pubs.opengroup.org/onlinepubs/9699919799/utilities/sort.html